spec/features/gardens/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'New Garden Page' do
+  describe 'a logged in user' do
+    before :each do
+      @user = User.new({id: 1,
+                      attributes: {
+                          email: '123@gmail.com' },
+                      relationships: {
+                          gardens: {
+                              data: [ { id: 1,
```

Comment:
```suggestion
                              data: [ { id: "1", type: "garden"} ]
```

Then you can remove all of the attributes below because they don't actually get passed from the backend